docs/modules/plugins/pages/plugin-mobile-app.adoc

```diff
@@ -475,4 +475,39 @@ When I change Appium session settings:
 |allowInvisibleElements|true |
 ----
 
+=== Select date in date picker
+
+==== *_Info_*
+
+Selects a next or previous picker wheel value in date picker
+
+[WARNING]
+Supported only for `iOS` platform
+Picker wheel argument must be `XCUIElementTypePickerWheel`
+
+==== *_Wording_*
+
+[source,gherkin]
+----
+When I select $direction value in picker wheel located `$locator`
+----
+
+==== Properties
```

Comment:
the properties should be placed in a separate section (not in the step)

Reply:
isn't it more convenient? we already did this for "Swipe to an element" step

Reply:
I believe for "swipe" it should be fixed as well
It could be more convenient if it's used once, but it's hard to maintain it, you can use cross-linking to refer the parts of the docs